local/version.go

```diff
@@ -0,0 +1,4 @@
+package local
+
+// Version of the local VPP implementation
+const Version = "20.09"
```

Comment:
It's actually `20.09-rc0`, since the VPP 20.09 has not been released yet and most certainly will have changes in the API.